src/App/integrationTests/collectRecords/benthicLit/App.NewBenthicAttributeBenthicLit.test.js

```diff
@@ -151,44 +157,47 @@ test('Benthic LIT observations add new benthic attribute - proposing new attribu
   const firstBenthicAttributeInput =
     within(observationsTable).getAllByDisplayValue('Dead Coral with Algae')[0]
 
-  userEvent.type(firstBenthicAttributeInput, 'supercalifragilistic')
+  await user.type(firstBenthicAttributeInput, 'supercalifragilistic')
 
   const noResultsButton = await screen.findByRole('button', {
     name: 'Propose New Benthic Attribute...',
   })
 
-  userEvent.click(noResultsButton)
+  await user.click(noResultsButton)
 
   const modal = screen.getByLabelText('Add New Benthic Attribute')
 
   const parentInput = await within(modal).findByLabelText('Parent')
   const nameInput = within(modal).getByLabelText('Name')
 
-  userEvent.type(parentInput, 'Dead')
+  await user.type(parentInput, 'Dead')
 
   const parentAutocompleteList = within(modal).getByRole('listbox')
 
   const deadCoralOption = within(modal).getByRole('option', {
     name: 'Dead Coral with Algae',
   })
 
-  userEvent.selectOptions(parentAutocompleteList, deadCoralOption)
+  await user.selectOptions(parentAutocompleteList, deadCoralOption)
 
-  userEvent.type(nameInput, 'Zosteraceae')
+  await user.type(nameInput, 'Zosteraceae')
 
   const nextScreenButton = await within(modal).findByRole('button', {
     name: 'Next',
   })
 
-  userEvent.click(nextScreenButton)
+  await user.click(nextScreenButton)
 
   const submitButton = await within(modal).findByRole('button', {
     name: 'Send to MERMAID for review',
   })
 
-  userEvent.click(submitButton)
+  expect(screen.getByLabelText('Add New Benthic Attribute'))
+  await user.click(submitButton)
 
-  await waitForElementToBeRemoved(() => screen.queryByLabelText('Add New Benthic Attribute'))
+  await waitFor(() =>
+    expect(screen.queryByLabelText('Add New Benthic Attribute')).not.toBeInTheDocument(),
+  )
```

Comment:
Sometimes `waitForElementToBeRemoved` fails when the whole test suite is run, but passes when the test is run on its own. I can make the issue go away by converting it to a `waitFor` with a `.not.toBeInTheDocument()`, however we probably need to be careful that we dont create false positives (in the exact same way we would have been careful for `waitForElementToBeRemoved` by asserting that the thing existed at all first, and then was removed as expected

Reply:
hmm maybe this will improve some of the tests timing out?

Reply:
It doesnt make the test suite on whole run faster unfortunately. I had hopes...

If anything the tests are now more finicky in my mind. Too many async micromanagements required to begin with. Now we seemingly have more (although maybe its not more, but just different async things)